docs/DafnyRef/Specifications.md

```diff
@@ -352,15 +352,36 @@ If the `FrameField` is not preceded by an expression then
 the frame expression is referring to that field of the current
 object. This form is only used within a method of a class or trait.
 
-The use of ``FrameField`` is discouraged as in practice it has not
-been shown to either be more concise or to perform better.
-Also, there's (unfortunately) no form of it for array
-elements---one could imagine
+A ``FrameField`` can be useful in the following case:
+When a method modifies only one field, rather than writing
 
 ```dafny
-  modifies a`[j]
+class A {
+  var i: int
+  var x0: int var x1: int var x2: int var x3: int var x4: int
```

Comment:
```suggestion
  var x0: int
  var x1: int
  var x2: int
  var x3: int
  var x4: int
```

And likewise below. I'd favour readability and good style over making the snippet compact.